create_infotexts.py

```diff
@@ -381,9 +374,114 @@ def process_depicted_place(self, place_string, places_mapping):
+        :return: string representing altered or unaltered description.
+        """
+        description = self.remove_svenska_cypernexpedition_from_description(item["Beskrivning"])
+
+        if not description.endswith("."):
+            description += "."
+
+        # step 1 in enrichment process
+        description_with_region = self.process_region_addition_to_description(
+            description,
+            item["Region, foto"],
+            item["Land, foto"]
+        )
+
+        # step 2 in enrichment process
+        description_with_keywords = self.process_keyword_addition_to_description(
+            description_with_region
+        )
+
+        # Step 3 in enrichment process
+
+        batch_description = ". Svenska Cypernexpeditionen 1927-1931."
+        full_description = description_with_keywords.rstrip(".") + batch_description
+
+        self.data["enriched_description"] = full_description
+
+    def process_keyword_addition_to_description(self, description_str):
+        """
+        Append keywords to description, unless the only keyword is "Svenska Cypernexpeditionen". 
+
+        :param description_str: string respresenting the value of column <Beskrivning>.
+        :return: string with added keywords, if existing 
+        """
+        if self.data["Nyckelord"]:
+            for kw in self.data["Nyckelord"]:
+                if not kw.lower() in description_str.lower():
+                    return self.append_keywords_to_desc(description_str, self.data["Nyckelord"])
+                else:
+                    return description_str
+        else:
+            return description_str
+
+    @staticmethod
+    def append_keywords_to_desc(description_str, kw_list):
+        """
+        Append a list of keywords to the existing <Beskrivning> field value.
+
+        :param description_str: string representing column <Beskrivning> in metadata.
+        :param kw_list: preprocessed list of keywords found in column <Nyckelord>.
+        :return: string representing concatenation of old <Beskrivning> plus keywords list.
+        """
+        newdesc = description_str
+        newdesc += " ''Nyckelord:'' "
+
+        for kw in kw_list:
+            newdesc += kw + ", "
+        return newdesc.rstrip(", ")
+
+    @staticmethod
+    def process_region_addition_to_description(description_str, region_str, country_str):
+        """
+        Add <Region, foto> to description string, except when it is already present, with some smartness.
+
+        :param description_str: String representing the processed/enriched description incl. <Nyckelord>.
+        :param region_str: String with the field value from column <Region ,foto> in metadata.
+        :param country_str: String with the field value from column <Land> in metadata. 
+        :return: String with possibly enriched description.
+        """
+        newdesc = description_str
+        if region_str:
+            newdesc += " ''Region:'' " + region_str
```

Comment:
OK, we used <br> in DecArch or GAR, even though I think it looks pretty 🦄  to have it all inline.